crates/ruff_benchmark/src/lib.rs

```diff
@@ -119,9 +119,9 @@ impl TestFile {
             Ok(TestFile::new(name.to_string(), content))
         } else {
             // File not yet cached, download and cache it in the target directory
-            let response = ureq::get(url.as_str()).call()?;
+            let mut response = ureq::get(url.as_str()).call()?;
 
-            let content = response.into_string()?;
+            let content = String::from_utf8_lossy(&response.body_mut().read_to_vec()?).to_string();
```

Comment:
could this just be

```suggestion
            let content = response.body_mut().read_to_string()?;
```

? Or are the semantics of `String::from_utf8_lossy()` important here?

Reply:
I haven't really debugged why but it throws an error:
```
thread 'main' panicked at crates/ruff_benchmark/benches/formatter.rs:45:42:
called `Result::unwrap()` on an `Err` value: Request(Io(Error { kind: InvalidData, message: "stream did not contain valid UTF-8" }))
```

which seems unexpected given that `read_to_string` should convert incorrect UTF-8 chars to `?` character.